### PR TITLE
[one-cmds] Add save_min_max option to one-quantize

### PR DESCRIPTION
**Pull request by jinevening**
This adds save_min_max option to one-quantize.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/11717

### CODE REVIEW

**Review by BalyshevArtem — APPROVED** [review]
LGTM

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -144,6 +144,12 @@ def _get_parser():
         help=
         "Force MaxPool Op to have the same input/output quantparams. NOTE: This option can degrade accuracy of some models.)"
     )
+    quantization_group.add_argument(
+        '--save_min_max',
+        action='store_true',
+        help=
+        "Save min/max of each tensor. NOTE: Min/max valuse are clipped according to calibration algorithms, such as percentile or moving average. Nudge adjustment is not applied."
```
Comment (seanshpark) [inline compiler/one-cmds/one-quantize:151]: can you plz split this line inside 100 cols?

Comment (seanshpark) [reply compiler/one-cmds/one-quantize:151]: this file has multiple lines that exceed 100 cols.
in another PR, it would be better to update them.

Comment (jinevening) [reply compiler/one-cmds/one-quantize:151]: Hmm.. It seems that formatting is not applied to one-quantize. Can I merge this as-is and update the format at once?

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**